Index: clone.pages.inc
===================================================================
--- clone.pages.inc	(revision 11)
+++ clone.pages.inc	(working copy)
@@ -163,7 +163,7 @@
       $node->book['mlid'] = NULL;
       $node->path = NULL;
       $node->files = array();
-      $node->title = t('Clone of !title', array('!title' => $node->title));
+      $node->title = $node->title;
 
       if (variable_get('clone_reset_'. $node->type, FALSE)) {
         $node_options = variable_get('node_options_'. $node->type, array('status', 'promote'));
